app.js

```diff
@@ -18,6 +18,9 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// Always run in UTC, no matter what
+process.env.TZ = 'UTC';
```

Comment:
This is only an issue when your machine isn't running in UTC (fwiw, all our servers are).
New date objects default to the `TZ` timezone when created. It becomes a real issue when serialising a calendar to ical (the cozy-ical library doesn't seem to handle this terribly well).

(fwiw, we had to do it in OAE as well: https://github.com/oaeproject/Hilary/blob/master/node_modules/oae-util/lib/oae.js#L62)